### PR TITLE
added code, need to write tests

### PR DESCRIPTION
**Pull request by SianCulligan**
- [x] Top-level README “Table of Contents” is updated
 - [x ] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

**Comment by SianCulligan** [comment]
- [ X] Top-level README “Table of Contents” is updated
 - [ X] Feature tasks for this challenge are completed
 - [ X] Unit tests written and passing
     - [ X] “Happy Path” - Expected outcome
     - [ X] Expected failure
     - [ X] Edge Case (if applicable/obvious)
 - [ X] README for this challenge is complete
     - [ X] Summary, Description, Approach & Efficiency, Solution
     - [ X] Link to code
     - [ X] Picture of whiteboard

File: code_challenges/401/fifoAnimalShelter/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+### Challenge
+Create a class called AnimalShelter which holds only dogs and cats. The shelter operates using a first-in, first-out approach.
+
+### Approach & Efficiency
+I approached the problem by first thinking through the basic problem & building the function to handle edge cases.
+
+
+### Solution
+[White-board](./assets/fifo.jpeg)</br></br>
```
Comment (lizkavalski) [inline code_challenges/401/fifoAnimalShelter/README.md:14]: try capitalizing the JPEG to see if the image will show.
